Scripts/Player_Scripts/player.gd

```diff
@@ -13,13 +19,45 @@ func _input(event):
 		
 func _ready():
 	Input.set_mouse_mode(Input.MOUSE_MODE_CAPTURED)
-	
 
 func _physics_process(delta: float) -> void:
 	_apply_gravity(delta)
 	_jump()
 	_handle_movement()
 	move_and_slide()
+	
+	# Sticking flashlight code here for now
```

Comment:
avaoid putting more than extremely simple code in process_or physics process, please move them to their own function